cmd/ocm-backplane/managedJob/logsManagedJob.go

```diff
@@ -101,6 +104,11 @@ func newLogsManagedJobCmd() *cobra.Command {
 				return err
 			}
 
+			err = deprecation.CheckResponseDeprecation(resp)
+			if errors.Is(err, deprecation.ErrDeprecation) {
+				logger.Warnf("The server indicated that this version of backplane-cli is deprecated, please update as soon as possible.")
```

Comment:
Duplicates error block, and this may be avoided with the previous comment

Reply:
Since we changed the plan so this only prints on `login` and `session` so this message is now only in one place.